domains/ploszukiwacz.json

```diff
@@ -6,6 +6,8 @@
     "email": "ploszukiwacz1@gmail.com"
   },
   "record": {
-    "CNAME": "ploszukiwaczdev.github.io"
+    "CNAME": "ploszukiwaczdev.github.io",
+    "MX": ["mx1.forwardemail.net", "mx2.forwardemail.net"],
+    "TXT": "forward-email=ploszukiwacz1@proton.me"
```

Comment:
You cannot use CNAME with any other records.

Reply:
Unresolved as issue still has not be fixed.